src/environment/storyteller.py

```diff
@@ -0,0 +1,35 @@
+import re
+from typing import List, Optional
+
+
+class Storyteller:
+    def __init__(self, atmosphere: List[str], generate_text):
+        self._generate_text = generate_text
+
+        self._system_prompt = f"""
+            S: You are a storyteller in a text-based game.
+            The game has {', '.join(atmosphere)} atmosphere. 
+            Your task is to describe locations in a light artistic style.
+            There may be or may not be additional descriptive keywords.
```

Comment:
What do you mean by this? may or may not? Shouldn't it be deterministic?

Reply:
Llama and other models are probabilistic. That means you won't get the same answer twice.

We solved this comment easing our prototype constraints by oral agreement. I will implement this in the following PR.